fuel/src/main/kotlin/com/github/kittinunf/fuel/util/FuelRouting.kt

```diff
@@ -26,6 +26,10 @@ interface FuelRouting: Fuel.RequestConvertible {
      * It uses a pair with String, Any.
      */
     val params: List<Pair<String, Any?>>?
+    /**
+     * Body to handle binary type of request (e.g. application/octet-stream )
+     */
+    val bytes: ByteArray?
```

Comment:
I guess this looks like a better way of sending the binary data. Do you think we should deprecate the `body` in the string version and prefer this instead @yhanada ?

Reply:
thanks for your comment @kittinunf 
Do you think that Fuel is used string format such as JSON more often to POST?
If so, it may be early to deprelate string version.
Should I fix it as follows?

```
    body?.let {
        encoder.request.body(it)
    } ?: bytes?.let {
        encoder.request.body(it)
    }
```

Reply:
Okay.

Reply:
Thanks I fixed it. 🙇